dev-requirements.txt

```diff
@@ -1,2 +1 @@
--r requirements.txt
-http://downloads.sourceforge.net/project/pyasn1/pyasn1/0.1.8/pyasn1-0.1.8rc2.tar.gz?r=http%3A%2F%2Fsourceforge.net%2Fprojects%2Fpyasn1%2Ffiles%2Fpyasn1%2F0.1.8%2F&ts=1426346728&use_mirror=heanet

```

Comment:
Ah, I did not realize pyasn1 is required for tests only. My bad.

Latest released version is 0.1.9 and it is on PyPI. So I believe we could do just pyasn1>=0.1.8.